### ISSUE TITLE
Add 'aria-expanded' and 'aria-haspopup` to accordion component

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6479]
When closed, the accordion triggers (`accordion__link`) should have:
- `aria-haspopup=true`
- `aria-expanded=false`

When open:
- `aria-haspopup=true`
- `aria-expanded=true`

**Comment by nlhkabu** [comment]
Alternative: http://heydonworks.com/practical_aria_examples/#progressive-collapsibles

uses `aria-expanded` and `aria-controls`.
